src/auth/providers.js

```diff
@@ -50,13 +44,12 @@ function userPassProvider(auth) {
   return {
     /**
      * Login to a stitch application using username and password authentication
-     *
-     * @param {String} username the username to use for authentication
-     * @param {String} password the password to use for authentication
-     * @returns {Promise}
      */
-    login: (username, password, opts) => {
+    authenticate: data => {
```

Comment:
Should we write a new JS doc for `authenticate`?

Reply:
After inlining the `({ username, password }) => ...` params, I added back the JS Doc lines describing the params.